modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -56,18 +58,28 @@ Default - `null`
 | `retention.local.target.bytes`  +
 No default
 
+| Segment lifetime
+| `log_segment_ms`  +
+No default
+| `segment.ms`  +
+No default
+
 | Segment size
 | `log_segment_size`  +
 Default - `1073741824`
 | `segment.bytes`  +
 No default
 |===
 
-Data expires from object storage following `retention.ms` or `retention.bytes`. For example, if `retention.bytes` is set to 10 GiB, then every partition in the topic has a limit of 10 GiB storage in the cloud. When `retention.bytes` is exceeded by data in object storage, the data in object storage is trimmed. With Tiered Storage enabled, data expires from local storage following `retention.local.target.ms` or `retention.local.target.bytes`.
+Data expires from object storage following both `retention.ms` and `retention.bytes`. For example, if `retention.bytes` is set to 10 GiB, then every partition in the topic has a limit of 10 GiB storage. When `retention.bytes` is exceeded by data in object storage, the data in object storage is trimmed, even if `retention.ms` is not yet exceeded. With Tiered Storage enabled, data expires from local storage following `retention.local.target.ms` or `retention.local.target.bytes`.
+
+image:shared:time-based-segment-rolling.png[Time-based segment rolling]
+
+As noted above, retention policy functions by deleting or compacting closed segments. The segment lifetime and segment size configurations help ensure new segments are created regularly within each partition. The above illustration shows the relationship between segments and the retention policy using time-based limits.  If these values are set too high, especially if they're higher than the corresponding retention values, the segment may fill available disk space before closing and therefore never become eligible for deletion or compaction. If the values are set too low, your partitions will have a large number of segments which must be checked each time deletion or compaction processes execute, having a potential adverse impact on system resource utilization.
```

Comment:
```suggestion
Retention policy functions by deleting or compacting closed segments. The segment lifetime and segment size configurations help ensure new segments are created regularly within each partition. This illustration shows the relationship between segments and the retention policy using time-based limits. If these values are set too high, especially if they're higher than the corresponding retention values, the segment may fill available disk space before closing and therefore never become eligible for deletion or compaction. If the values are set too low, your partitions will have a large number of segments that must be checked each time deletion or compaction processes execute, having a potential adverse impact on system resource utilization.
```
I was taught for accessibility (screen readers) never to use navigations in doc like above/below.

Reply:
Can you get Ben Weinstein to make this image consistent with our image standards? I have trouble understanding what it's trying to show. And I hate the split case "Message & Log segment rolling based on Time"

Reply:
I snagged the image from Travis, let me see if we can get it tidied up!

Reply:
Image replaced and language revised to focus on how segments roll and how that impacts retention.